### ISSUE TITLE
How to train on my own dataset?

### ISSUE THREAD

**Issue by LonglongaaaGo** [vitoralbiero/img2pose#5]
Hellow!
Thanks for your perfect project!
I wonder to know how to train on my own dataset!
I notice that you use the five landmarks to generate 6DoF pose labels by using standard means. Can you share the code about these methods? 
Thanks for your beautiful work!
I am waiting for your reply！

**Comment by vitoralbiero** [comment]
Hello, thanks for your interest in our work.
To train with your own dataset, if you have manually annotated points you can store them into JSON files following the structure of the [WIDER FACE annotations](https://github.com/vitoralbiero/img2pose/wiki/Annotations).
Or you can use can follow the newly added [instructions](https://github.com/vitoralbiero/img2pose/blob/main/README.md#training-on-your-own-dataset) to annotate your own dataset

Hope this helps!

**Comment by LonglongaaaGo** [comment]
Thank you a lot!
But I find that I need the pre_trained Retinaface model. So, is the model from "https://github.com/biubug6/Pytorch_Retinaface"?

**Comment by vitoralbiero** [comment]
I used this implementation https://github.com/deepinsight/insightface/tree/master/detection/RetinaFace
The pre-trained model used was downloaded from https://www.dropbox.com/s/53ftnlarhyrpkg2/retinaface-R50.zip?dl=0

**Comment by LonglongaaaGo** [comment]
Thank you for your reply！
Recently, I try to generate annotations on the celeb dataset by using the method recommended in your early reply. But I don't know why these masks I generated do not align with a face.  When I use the render_plot function on an annotation, the mask is not aligned with the corresponding picture's face.
I just change the max_size=500 and min_size=256 in annotate_dataset.py because I want to generate annotations on the celeb -256 dataset with pixel size 256*256.


> I used this implementation https://github.com/deepinsight/insightface/tree/master/detection/RetinaFace
> The pre-trained model used was downloaded from https://www.dropbox.com/s/53ftnlarhyrpkg2/retinaface-R50.zip?dl=0
